config-connector/solutions/iam/kpt/pubsub-topic/README.md

```diff
@@ -0,0 +1,56 @@
+Pubsub Topic
+==================================================
+
+# NAME
+
+  pubsub-topic
+
+# SYNOPSIS
+
+  Config Connector compatible YAML files to grant a role to a particular IAM member for a PubSub topic.
+
+# CONSUMPTION
+
+  Using [kpt](https://googlecontainertools.github.io/kpt/):
+
+  ```
+  BASE=https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git
+  kpt pkg get $BASE/config-connector/solutions/iam/kpt/pubsub-topic pubsub-topic
+  ```
+
+# REQUIREMENTS
+
+  A cluster with Config Connector installed managing a GCP project with the
```

Comment:
This sounds like a cluster is managing a GCP project and I think this doesn't really describe the relationship properly.

Maybe something like "Config Connector successfully installed in a K8s cluster and the project managed by Config Connector has Pub/Sub API enabled"?